src/domains/Booking/components/BookingsByType/BookingsByType.js

```diff
@@ -0,0 +1,82 @@
+import { BOOKINGS_FETCH_LIMIT, COLLECTIONS } from '__constants__'
+import { Col, Container, Row, Text } from '@qonsoll/react-design'
+import { memo, useMemo } from 'react'
+import { useBookings, useGetBookingsCounter } from '../../../Booking/hooks/get'
+
+import BookingList from '../BookingList'
+import PropTypes from 'prop-types'
+import { useTranslations } from 'contexts/Translation'
+
+const { BOOKINGS } = COLLECTIONS
+function BookingsByType(props) {
+  const {
+    additionalQuery,
+    bookingsRef,
+    bookingTitle,
+    bookingsCounterName,
+    bookingsBelongTo
+  } = props
+
+  const { t } = useTranslations()
+  const computetBookingsRef = useMemo(() => {
```

Comment:
this part should be incapsulated inside useBookings hook

same with loadMoreAvailable variable below

Reply:
done